### PR TITLE
Add label to entries list checkboxes

### PR DESCRIPTION
**Pull request by AbdiTolesa**
Fix https://github.com/Strategy11/formidable-pro/issues/5239

CleanShot 2024-08-05 at 16 52 47@2x

### CODE REVIEW

File: classes/helpers/FrmEntriesListHelper.php [hunk 1]
```diff
@@ -292,6 +292,8 @@ public function single_row( $item, $style = '' ) {
 
 		// Set up the checkbox ( because the user is editable, otherwise its empty )
 		$checkbox = "<input type='checkbox' name='item-action[]' id='cb-item-action-{$item->id}' value='{$item->id}' />";
+		/* translators: %s: Form name */
+		$checkbox .= "<label for='cb-item-action-{$item->id}'/><span class='screen-reader-text'>" . sprintf( __( 'Select %s', 'formidable' ), FrmFormsHelper::edit_form_link_label( $item->form_id ) ) . '</span></label>';
```
Comment (truongwp) [inline classes/helpers/FrmEntriesListHelper.php:296]: There is a `/>` in the label tag, which is wrong.

Comment (AbdiTolesa) [reply classes/helpers/FrmEntriesListHelper.php]: I think `screen-reader-text` should be kept as is to be consistent with how core WP does it.

**Review by truongwp — APPROVED** [review]
This looks good. Thanks @AbdiTolesa!

**Review by Crabcyborg — APPROVED** [review]
Thanks @AbdiTolesa!

I added some escaping to the translations.

I also removed the Cypress rule that was breaking.

I think we're good to merge this now.

🚀

**Pull request merged**